inductiva/commands/__init__.py

```diff
@@ -0,0 +1,2 @@
+# pylint: disable=missing-module-docstring
+from .commands import Command
```

Comment:
I don't like having `__init__` exporting everything.
Just leave it empty for now.

Reply:
Okay, it is the reverse. We should expose it here, and not on the main `__init__.py`.

So that the above code works.